### ISSUE TITLE
Generate document describing elementary calculations for specular reflectometry

### ISSUE THREAD

**Issue by arm61** [reflectivity/reproducibility#12]
This would be a webpage, and would like with the Education and Outreach WG.

**Comment by arm61** [comment]
This is potentially linked to the paper being worked on in the education and outreach WG. @tomarnoldess what do you think?

**Comment by christykinane** [comment]
Hi Andrew, yes to be honest there is big overlap with the EaOWG. We need to talk to them a coordiante a bit more.

**Comment by tomarnoldess** [comment]
Yes I agree there is overlap, but i wasn't involved in the original discussion on this. It sounds slightly different in that the publication is describing some more complex stuff for people new to the field, where as this is a tool to ensure consistency for people writing code. Or have i misunderstood?

**Comment by arm61** [comment]
Testing my memory. But I think it was @aglavic that brought it up in one of the sessions.

**Comment by aglavic** [comment]
Not sure about this specific point, maybe it was part of the discussion about the best practice in commissioning an instrument.

I agree, this has a lot of overlap to EaOWG. It is also a basis for anything we collect about theory in general, as many aspects of these calculations are scattered throughout several publications, which all use slightly different definitions and terms. So if we collect it centrally on our website all of these formulae/algorithms can be described with the same basis, making future implementations much easier.

**Comment by tomarnoldess** [comment]
OK i suggest you all come to the EaO session we will have in this... i will schedule this first, on 16th Sept and then we can discuss how this relates and what should go on the website before we have the session on the website. @aglavic, you didn't respond to the doodle about EaO, are you available on the 16th?

**Comment by aglavic** [comment]
@tomarnoldess I don't think I got any invitation for that, but I think I'm not on the EaO newsletter. (I'm easily getting lost in the flood of github emails and I have no idea how to check if I'm subscribed to the newsletter or check older version as I didn't find any news history on the Website, sorry.)
Besides the instrument scientist meeting at 10:30 I should be free on the 16th.

**Comment by arm61** [comment]
Just as a heads up @aglavic, you can reduce the emails here https://github.com/settings/notifications

**Comment by tomarnoldess** [comment]
@aglavic the news emails etc are through the mailchimp mailing list. You have to have signed up to the right mailing list to get notified about each of the working groups activities, not just the general mailing list. It is independent of the github emails. I'll invite you to the meeting when i get around to scheduling it.
